package.json

```diff
@@ -11,16 +11,18 @@
     "lint-python": "flake8 webapp tests && black --check --line-length 79 webapp tests",
     "lint-scss": "sass-lint static/**/*.scss --verbose --no-exit",
     "test-python": "python3 -m unittest discover tests",
-    "serve": "./entrypoint 0.0.0.0:${PORT}",
+    "serve": "./entrypoint $(hostname -I | awk '{print $1;}'):${PORT}",
```

Comment:
I'm not sure about this, the `0.0.0.0` was intentional. This `hostname` usage also makes the `serve` target only work on Ubuntu and not Mac OS. Effectively making `dotrun` a requirement.

Reply:
Yeah I'm not quite sure about it either. Alright, let's stop doing this. I'll remove it from the advice in the dotrun readme.

Reply:
See https://github.com/canonical-web-and-design/dotrun/commit/074aa2afa3a3122e02ee167efc94a1f8df035dd7